css/wallscreens/styles.css

```diff
@@ -23,6 +24,9 @@ pre {
   text-transform: lowercase;
 }
 
+.fas {
```

Comment:
```suggestion
button .fas,
.button .fas  {
```

Scope this to just button-y things?

Reply:
That won't catch the previous and next buttons (at least given the current HTML). Plus I don't expect us to use icons anywhere else in the UI. But I'm happy to update things, depending on how you want to deal with the previous and next buttons.

Reply:
I think those have the `.button` class?

Reply:
We could do `button .fas, nav .fas` or `button .fas, a .fas`. That just felt like it was making things more complicated than necessary. But it would be more specific to our current cases.